docs/04-community/02-rspack.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: Rspack Integration
+nav_title: Rspack
+description: Using the next-rspack plugin to use Rspack as bundler
+---
+
+The Rspack team has created a community plugin for Next.js.
```

Comment:
Maybe tag this along
The Rspack team has created a community plugin for Next.js, which is part of a [partnering effort](https://rspack.rs/blog/rspack-next-partner) with the Rspack team